Dockerfile

```diff
@@ -9,84 +9,34 @@ RUN mkdir -p /opt/app/bin/
 
 # Install packages
 RUN yum update -y
-RUN amazon-linux-extras install epel -y
-RUN yum install -y cpio yum-utils tar.x86_64 gzip zip python3-pip binutils
+RUN yum install -y cpio clamav clamav-update yum-utils tar.x86_64 gzip zip python3.11 python3.11-pip binutils
 
 # This had --no-cache-dir, tracing through multiple tickets led to a problem in wheel
-RUN pip3 install -r requirements.txt
+RUN pip3.11 install -r requirements.txt
 RUN rm -rf /root/.cache/pip
 
-# Download libraries we need to run in lambda
-WORKDIR /tmp
-RUN yumdownloader -x \*i686 --archlist=x86_64 clamav
-RUN rpm2cpio clamav-0*.rpm | cpio -vimd
-
-RUN yumdownloader -x \*i686 --archlist=x86_64 clamav-lib
-RUN rpm2cpio clamav-lib*.rpm | cpio -vimd
-
-RUN yumdownloader -x \*i686 --archlist=x86_64 clamav-update
-RUN rpm2cpio clamav-update*.rpm | cpio -vimd
-
-RUN yumdownloader -x \*i686 --archlist=x86_64 json-c
-RUN rpm2cpio json-c*.rpm | cpio -vimd
-
-RUN yumdownloader -x \*i686 --archlist=x86_64 pcre2
-RUN rpm2cpio pcre*.rpm | cpio -vimd
-
-RUN yumdownloader -x \*i686 --archlist=x86_64 libtool-ltdl
-RUN rpm2cpio libtool-ltdl*.rpm | cpio -vimd
-
-RUN yumdownloader -x \*i686 --archlist=x86_64 libxml2
-RUN rpm2cpio libxml2*.rpm | cpio -vimd
-
-RUN yumdownloader -x \*i686 --archlist=x86_64 bzip2-libs
-RUN rpm2cpio bzip2-libs*.rpm | cpio -vimd
-
-RUN yumdownloader -x \*i686 --archlist=x86_64 xz-libs
-RUN rpm2cpio xz-libs*.rpm | cpio -vimd
-
-RUN yumdownloader -x \*i686 --archlist=x86_64 libprelude
-RUN rpm2cpio libprelude*.rpm | cpio -vimd
-
-RUN yumdownloader -x \*i686 --archlist=x86_64 gnutls
-RUN rpm2cpio gnutls*.rpm | cpio -vimd
-
-RUN yumdownloader -x \*i686 --archlist=x86_64 nettle
-RUN rpm2cpio nettle*.rpm | cpio -vimd
-
-
 # Copy over the binaries and libraries
-RUN cp /tmp/usr/bin/clamscan /tmp/usr/bin/freshclam /tmp/usr/lib64/* /usr/lib64/libpcre.so.1 /opt/app/bin/
-RUN cp /bin/ld /usr/lib64/libbfd-*.amzn2.so /opt/app/bin/
+RUN cp /bin/clamscan /usr/bin/freshclam /bin/ld /opt/app/bin/.
 
-# Copy shared libraries required for freshclams to run in lambda
-RUN cp /usr/lib64/libcurl.so.4 /usr/lib64/libnghttp2.so.14 /usr/lib64/libidn2.so.0 /usr/lib64/libssh2.so.1 \
-/usr/lib64/libldap-2.4.so.2 /usr/lib64/liblber-2.4.so.2 /usr/lib64/libunistring.so.0 /usr/lib64/libsasl2.so.3 \
-/usr/lib64/libssl3.so /usr/lib64/libsmime3.so /usr/lib64/libnss3.so /usr/lib64/libcrypt.so.1 /opt/app/bin/
+# Copy shared libraries required to run in lambda
+RUN bash /copylibs.sh
 
 # Fix the freshclam.conf settings
 RUN echo "DatabaseMirror database.clamav.net" > /opt/app/bin/freshclam.conf
 RUN echo "CompressLocalDatabase yes" >> /opt/app/bin/freshclam.conf
 RUN echo "ScriptedUpdates no" >> /opt/app/bin/freshclam.conf
 RUN echo "DatabaseDirectory /var/lib/clamav" >> /opt/app/bin/freshclam.conf
 
-RUN yum install shadow-utils.x86_64 -y
-
-RUN groupadd clamav
-RUN useradd -g clamav -s /bin/false -c "Clam Antivirus" clamav
-RUN useradd -g clamav -s /bin/false -c "Clam Antivirus" clamupdate
-
```

Comment:
Since we are packaging things up into a zip file and not deploying the Docker image to lambda the user/group does not carry over.